pkg/webhook/plugins/requirenodewithfuse/require_node_with_fuse.go

```diff
@@ -0,0 +1,75 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package requirenodewithfuse
+
+import (
+	"github.com/fluid-cloudnative/fluid/pkg/ddc/base"
+	"github.com/fluid-cloudnative/fluid/pkg/utils"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+/*
+   This plugin is for pods with a  dataset.
+   They should require nods with fuse.
+*/
+
+const NAME = "RequireNodeWithFuse"
+
+type RequireNodeWithFuse struct {
+	client client.Client
+	name   string
+}
+
+func NewPlugin(c client.Client) *RequireNodeWithFuse {
+	return &RequireNodeWithFuse{
+		client: c,
+		name:   NAME,
+	}
+}
+
+func (p *RequireNodeWithFuse) GetName() string {
+	return p.name
+}
+
+func (p *RequireNodeWithFuse) Mutate(pod *corev1.Pod, runtimeInfos []base.RuntimeInfoInterface) (shouldStop bool) {
```

Comment:
`shouldStop` is not assigned, only have an initial value
is this expected logic ?

Reply:
Yeah, the value is false by design.